README.md

```diff
@@ -135,7 +135,7 @@ These options apply to all APIs.
 
 ### displayCheck option
 
-Type: `full` | `non-zero-area` | `none` . Default: `full`.
+Type: `full` | `full-with-hidden` | `non-zero-area` | `none` . Default: `full`.
```

Comment:
I thought the new name was `legacy-full`?

Reply:
Yes, it's `legacy-full`. Thanks for catching that. Will fix it right away.